src/ResourceManager/GuestConfiguration/Commands.GuestConfiguration/help/Get-AzureRmVMGuestPolicyStatusHistory.md

```diff
@@ -0,0 +1,204 @@
+---
+external help file: Microsoft.Azure.Commands.GuestConfiguration.dll-Help.xml
+Module Name: AzureRM.GuestConfiguration
+online version: https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/GuestConfiguration/Commands.GuestConfiguration/help/Get-AzureRmVMGuestPolicyStatusHistory.md
+schema: 2.0.0
+---
+
+# Get-AzureRmVMGuestPolicyStatusHistory
+
+## SYNOPSIS
+Gets guest configuration policy compliance status history for an initiative of type "Guest Configuration" that is assigned to a VM.
+An initiative is a policy of definition type "Initiative".
+
+## SYNTAX
+
+### VmScope (Default)
+```
+Get-AzureRmVMGuestPolicyStatusHistory [-ResourceGroupName] <String> [-VMName] <String> [-ShowOnlyChanges]
```

Comment:
Please follow instructions here to suppress signature issues: https://github.com/Azure/azure-powershell/blob/preview/documentation/Debugging-StaticAnalysis-Errors.md

Reply:
changed. could you check ? Is this what needed

Reply:
fixed